samza-test/src/test/java/org/apache/samza/test/operator/StreamApplicationIntegrationTestHarness.java

```diff
@@ -98,8 +100,9 @@
 public class StreamApplicationIntegrationTestHarness extends AbstractIntegrationTestHarness {
   private KafkaProducer producer;
   private KafkaConsumer consumer;
+  protected KafkaSystemAdmin systemAdmin;
   private StreamApplication app;
-  private ApplicationRunner runner;
+  protected RemoteApplicationRunner runner;
```

Comment:
I am using `RemoteApplicationRunner` here because I need to call `RemoteApplicationRunner.getIntermediateStreams()`. Due to the existing limit in Samza test suite, we have to use `RemoteApplicationRunner` instead of `LocalApplicationRunner`. Prateek left comment in `StreamApplicationIntegrationTestHarness` that says "We can't use LocalApplicationRunner since ZkJobCoordinator doesn't currently create". I think Prateek can explain more deeply why we have to use `RemoteApplicationRunner`.